pkg/pipelines/tekton/validate.go

```diff
@@ -32,17 +31,6 @@ func validatePipeline(f fn.Function) (string, error) {
 			return "", ErrRuntimeRequired
```

Comment:
think now we can do away with using `warningMsg` in this function? https://github.com/knative/func/blob/b2f9dc538870f552ca2bc7393a88e3660c54a4c3/pkg/pipelines/tekton/validate.go#L27
and also its usage here:
https://github.com/knative/func/blob/b2f9dc538870f552ca2bc7393a88e3660c54a4c3/pkg/pipelines/tekton/pipelines_provider.go#L121
https://github.com/knative/func/blob/b2f9dc538870f552ca2bc7393a88e3660c54a4c3/pkg/pipelines/tekton/pipelines_pac_provider.go#L33